packages/form-field/src/FormFieldInputContainer/FormFieldInputContainer.styles.ts

```diff
@@ -329,40 +362,30 @@ export const iconsWrapperStyles = css`
   gap: ${spacing[1]}px;
 `;
 
-export const iconStyles: Record<Theme, string> = {
+export const iconDisabledStyles: Record<Theme, string> = {
```

Comment:
similar to above, I removed the `disabled` attribute from the associated icon in favor of applying this classname if `disabled` prop is true

Reply:
Nit: I think a benefit of keeping `aria-disabled` in the CSS is that it's easier to see how these styles override other styles in dev tools; it's not a deal breaker, though.

Reply:
I hear you on this. It would have been easier to keep the styles as-is but felt more correct based off [MDN associated roles](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-disabled#associated_roles) to apply the `aria-disabled` attribute to the input tag itself